requirements-tests.txt

```
@@ -6,7 +6,7 @@ mypy ==1.8.0
 dirty-equals ==0.8.0
 sqlmodel==0.0.22
 flask >=1.1.2,<4.0.0
-anyio[trio] >=3.2.1,<4.0.0
+anyio[trio] >=3.2.1,<5.0.0
```

starlette already pins this
```suggestion
anyio[trio] >=3.4.0,<5.0.0
```